requirements.txt

```diff
@@ -3,13 +3,13 @@ docopt == 0.6.2
 mediawiki-utilities==0.4.14
 mwparserfromhell == 0.3.3
 nltk == 3.0.1
-nose == 1.3.4
-numpy == 1.9.2
+nose >= 1.3.4
+numpy >= 1.9.2
 pytz == 2015.4
 requests >= 2.0.0, < 3.0.0
-setuptools == 15.1
+setuptools >= 15.1
 pyenchant == 1.6.6
-scipy == 0.14.0
-scikit-learn == 0.15.2
+scipy >= 0.14.0
+scikit-learn >= 0.15.2
```

Comment:
scipy is locked at 0.14.0 because that's what we get from the repos with Debian Jessie

If we switch to scikit-learn 0.16.1, we'll need to re-generate our models.  That shouldn't be preventatively difficult, but I don't want to do it unnecessarily.  scikit-learn is our most important dependency to freeze.